docs/index.rst

```diff
@@ -86,6 +86,10 @@ used through the ``--static-path`` and ``--templates-path`` parameters.
 
 Gits uses
 
+* `SSH <https://en.wikipedia.org/wiki/Secure_Shell>`_ to remotely login into a 
+  system. You need to ensure, that a ssh demon is running, before you'll start
```

Comment:
@eugulixes, I hope to merge it today, to ensure, that tomorrow it'll be released.